pvm/ts/src/engine/texasHoldem-action-index.test.ts

```
@@ -21,101 +21,101 @@ describe("Texas Holdem - Turn Index", () => {
         game = TexasHoldemGame.fromJson(baseGameConfig, gameOptions);
     });
 
-    describe("Turn Index Initialization", () => {
-        it.only("should initialize with turn index of 0", () => {
+    describe("Action Index Initialization", () => {
+        it.only("should initialize with turn index of 1", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)